### PR TITLE
Enable users to navigate between Pathways and Practices

### LINKED ISSUE #1259: Enable users to navigate between Pathways and Practices
**What needs to be done? Why does it need to be done?**
Users should be able to navigate easily between Pathways and Practices. To achieve this, we will introduce a left sidebar that will allow users to navigate between Pathways and Practices, and will include links to global pages like methodology and user profile. 

Go to [section in PRD →](https://docs.google.com/document/d/1IvWek2k1qfrfJLCWB1kijUihcln-LKH2e8-WQVHCcFo/edit#heading=h.5brxap3xkn3c)
Go to [mocks in Figma →](https://www.figma.com/file/Mw6kSEP2jdr9i7UVEjPuCI/UP-Consolidation?node-id=1420%3A9457) 
Go to [prototype in Figma →](https://www.figma.com/proto/Mw6kSEP2jdr9i7UVEjPuCI/UP-Consolidation?page-id=776%3A2729&node-id=1150%3A3256&viewport=250%2C48%2C0.14&scaling=scale-down-width&starting-point-node-id=1420%3A9457&hide-ui=1)

Additional information:
- The new left sidebar component should be named `ViewNavigation`, and be added to the `PathwaysLayout` component.
- The "Practices" icon should link to `/community/practices`, and The "Pathways" icon should link to `/pathways`.
- The Methodology icon will be added as part of another ticket - no need to add it as part of this work.
- The user icon will be moved from the top right corner nav, to the bottom left as part of another ticket - no need to move it as part of this work.

The `PathwaysLayout` is only available when the tenant is set to US_ID, so when you run the app locally using `yarn offline`, click on the profile icon in the upper right hand corner, select Idaho, then View Dashboard. Then manually change your path to `localhost:3000/pathways`. You should see the navigation skeleton on this page. 

Please use the colors and fonts from `CoreConstants.scss`. These are our standardized styles across Recidiviz products. Let Jen know if you need a color or font that is not in that file. Thank you!

**Note:**
Product names, icons, and descriptions are still being finalized. Feel free to use the icons and copy in the mocks as placeholders for now.

### PR DESCRIPTION
**Pull request by nasaownsky**
## Description of the change

Added `ViewNavigation` component

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #1259 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally
- [ ] E2E have been run for Lantern changes ([Steps in the Readme](https://github.com/Recidiviz/pulse-dashboard#running-e2e-tests))

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/core/ViewNavigation/ViewNavigation.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          activeClassName="ViewNavigation--active"
+          className="ViewNavigation__navlink"
+          to="/community/practices"
+        >
+          <PracticesLogo />
+        </NavLink>
+        <div className="ViewNavigation__tooltip">
+          <h5 className="ViewNavigation__tooltip-header">Practices</h5>
+          <div className="ViewNavigation__tooltip-body">
+            A birds-eye view of staff- and region-level resources and
```
Comment (jovergaag) [inline src/core/ViewNavigation/ViewNavigation.tsx:56]: I see that the "staff-" is in the designs, but I think that's a mistake. Checking with the designer...

File: src/core/PathwaysLayout.scss [hunk 2]
```diff
@@ -18,12 +18,20 @@
...
 
 .PathwaysLayout {
+  display: flex;
+  min-height: 100vh;
+
+  &__main {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    margin-left: 5rem;
```
Comment (jovergaag) [inline src/core/PathwaysLayout.scss:28]: We need the `position: fixed;` declaration here to help position the `IE11Banner`

File: src/assets/static/images/practices.svg [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (jovergaag) [inline src/assets/static/images/practices.svg:1]: There's something not exactly with this SVG. There is a small discoloration inside of the dot. Can you try to re-create the SVG to see if it looks any better?

Screen Shot 2021-09-10 at 3 19 21 PM

**Review by jovergaag — CHANGES_REQUESTED** [review]
Hi @nasaownsky, great first PR! It looks amazing.

Just a couple of small comments for some finishing touches.

In addition to the comments below, can you please add ` background-color: $marble-3;` to the `.IE11Banner` high level block in `IE11Banner.scss`. The header in on the Pathways view is slightly different than on the Core view, so we're getting a small white strip. Instead of playing with the padding across both views, let's just change the background color so it is invisible.

```
.IE11Banner {
  background-color: $marble-3;
...
}

File: src/assets/static/images/practices.svg (hunk 3 above)
Comment (nasaownsky) [reply src/assets/static/images/practices.svg:1]: Fixed that.

File: src/core/PathwaysLayout.scss (hunk 2 above)
Comment (nasaownsky) [reply src/core/PathwaysLayout.scss:28]: The two adjacent fixed elements kinda breaks the header, so my suggestion is to modify `IE11Banner ` styles if its in the `PathwayLayout` component. Tested this in IE11 and its working as expected.

File: src/assets/static/images/practices.svg (hunk 3 above)
Comment (jovergaag) [reply src/assets/static/images/practices.svg:1]: Looks great now, thank you!

File: src/core/ViewNavigation/ViewNavigation.tsx (hunk 1 above)
Comment (jovergaag) [reply src/core/ViewNavigation/ViewNavigation.tsx:56]: It sounds like this is intentional, so leave it as is! Thank you.

File: src/core/PathwaysLayout.scss (hunk 2 above)
Comment (jovergaag) [reply src/core/PathwaysLayout.scss:28]: Agreed, looks great!

**Review by jovergaag — CHANGES_REQUESTED** [review]
I found two other issues, apologies for not catching them during the first round.

1. When clicking on the Pathways icon, the tooltip stays open until the user clicks elsewhere on the page. We want the tooltip to display on hover only.
2. This new LH nav bar is visible on the Profile page. We want it to only be visible on the Pathways page, for now. Eventually it will be visible on the Profile page, but not until we are ready to launch the new view.

**Comment by nasaownsky** [comment]
Yeah, forgot to push fix for the first issue. 

As for the second - seems like this happens due to routing\data structure in `App.js`. I see that `PathwaysLayout` and `CoreLayout` uses same `CORE_TENANTS` and it somehow leads to displaying two layouts at the same time. Could you take a look at this? I poorly understand data structure in pulse for now. Thanks!

**Review by jovergaag — APPROVED** [review]
Looks great! Thank you @nasaownsky

**Pull request merged**